.github/workflows/build.yml

```diff
@@ -8,27 +8,29 @@ env:
 
 jobs:
   build-web:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
For the web version the Ubuntu version shouldn't matter, it's only about binaries